bootstrap/manifest_template.json

```diff
@@ -131,6 +131,34 @@
     "data": "{{ getenv \"CONTROLLER_KEY\" }}"
   },
   {
+    "id": "redis-api",
+    "app": {
+      "name": "redis-api",
+      "meta": {"flynn-system-app": "true"}
+    },
+    "action": "run-app",
+    "release": {
+      "env": {
+        "FLYNN_REDIS": "redis",
+        "CONTROLLER_KEY": "{{ (index .StepData \"controller-key\").Data }}",
+        "REDIS_IMAGE_URI": "$image_repository?name=flynn/redis&id=$image_id[redis]"
+      },
+      "processes": {
+        "web": {
+          "ports": [{"port": 80, "proto": "tcp"}],
+          "cmd": ["api"]
+        }
+      }
+    },
+    "artifact": {
+      "type": "docker",
+      "uri": "$image_repository?name=flynn/redis&id=$image_id[redis]"
+    },
+    "processes": {
+      "web": 2
+    }
+  },
+  {
```

Comment:
I tried moving `redis-api` and `redis-api-wait` down below the controller and I'm getting strange results. Here's the manifest file:

https://gist.github.com/benbjohnson/3b22c55f0bf1b30e119d

I'm seeing `deploy-app redis-api` during bootstrap but then it hangs on `wait redis-api-wait`. I checked the logs and I don't see any `redis-api` process running.

https://gist.github.com/anonymous/37681124ae54ac1738c8